src/gui/systray.cpp

```diff
@@ -176,7 +176,7 @@ void Systray::setupContextMenu()
     if (AccountManager::instance()->accounts().isEmpty()) {
         _contextMenu->addAction(tr("Add account"), this, &Systray::openAccountWizard);
     } else {
-        _contextMenu->addAction(tr("Open main dialog"), this, [this]{ showWindow(); });
+        _contextMenu->addAction(tr("Open Nextcloud Desktop"), this, [this]{ showWindow(); });
```

Comment:
for branding reasons these strings should set the app name through a parameter

```suggestion
        _contextMenu->addAction(tr("Open %1 Desktop").arg(Theme::instance()->appNameGUI()), this, [this]{ showWindow(); });
```

Reply:
thanks
very good catch